pySDC/tests/test_sweepers/test_MPI_sweeper.py

```diff
@@ -158,8 +176,8 @@ def test_sweeper(num_nodes, quad_type, residual_type, imex, initGuess, launch=Tr
 @pytest.mark.parametrize("quad_type", ['GAUSS', 'RADAU-RIGHT'])
 @pytest.mark.parametrize("residual_type", ['last_abs', 'full_rel'])
 @pytest.mark.parametrize("imex", [False])
-@pytest.mark.parametrize("initGuess", ['spread', 'copy', 'zero'])
-def test_sweeper_NCCL(num_nodes, quad_type, residual_type, imex, initGuess, launch=True):
+@pytest.mark.parametrize("init_guess", ['spread', 'copy', 'zero'])
```

Comment:
sensing some resistance about camelCase here ... :innocent:

Reply:
It just looked very inconsistent 🙈